inttests/GOSCALEIO_TEST.env

```diff
@@ -0,0 +1,9 @@
+# Make sure system has a name (not just an ID) for tests to pass
+GOSCALEIO_INSECURE=true
+GOSCALEIO_HTTP_TIMEOUT=60
+GOSCALEIO_ENDPOINT=https://10.60.156.69/
```

Comment:
we shouldn't upload internal IPs / specifics to public repos

Reply:
Updated